### PR TITLE
Add note about GCP_KEY in Readme

### PR DESCRIPTION
**Pull request by Eusebiotrigo**

### CODE REVIEW

**Comment by froblesmartin** [comment]
@Eusebiotrigo is it needed to trigger a new release just due to a change in the README? 🤔 If not, the update would not appear on the npmjs.com website, right?

**Review by froblesmartin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ GCP_USER: Your Google Cloud Platform user (email) associated with the project.
 GCP_KEY: The path to your GCP service account key file (JSON file).
 GCP_PROJECT_NAME: The name of your GCP project where Secrets Manager will be used.
 
+### IMPORTANT
+The GCP_KEY in the format of -----BEGIN PRIVATE KEY----- .... -----END PRIVATE KEY----- taken from the JSON
```
Comment (froblesmartin) [inline README.md:27]: Maybe worth mentioning that Cloudflare will then transform the newlines into `\n`?

Comment (Eusebiotrigo) [reply README.md:27]: Yes. We need to re-redact the Readme in that regard.

File: README.md [hunk 2]
```diff
@@ -1,63 +1,80 @@
...
+Integrating Google Secret Manager with Cloudflare Workers to securely manage and dynamically retrieve sensitive information at the edge with
+0 dependencies.
 
 ## Prerequisites
 
 Before you begin, make sure you have the following:
 
 - Node.js installed (https://nodejs.org/)
 - Google Cloud Platform (GCP) account with a project set up
+- Google Secrets Manager active in your account.
```
Comment (froblesmartin) [inline README.md:12]:
```suggestion
- Google Secrets Manager API active in your project.
```

File: README.md [hunk 3]
```diff
@@ -1,63 +1,80 @@
...
+    - secretmanager.secrets.create
+    - secretmanager.secrets.setIamPolicy
+    - secretmanager.secrets.update
+    - secretmanager.versions.add
+    - secretmanager.versions.access
+
+
+2. The `key` is the Private Key. This key can be obtained from GCP Console. IAM - Select User - Keys. Then, generate a key if it does
+   not exist
+   and take the privateKey field from the json.
```
Comment (froblesmartin) [inline README.md:41]:
```suggestion
2. The `key` is the Private Key. To obtain this key, follow [the official Google documentation](https://cloud.google.com/iam/docs/keys-create-delete#creating)
```

**Review by froblesmartin — APPROVED** [review]

**Pull request merged**